brave/brave-core
# Show "Add Media From Page" button in the playlist web ui

This button will show up only when we find media on active tab and don't have saved items. Clicking this button will show up PlaylistBubble that let users choose media items to add.


Resolves https://github.com/brave/brave-browser/issues/40231


https://github.com/user-attachments/assets/ed907f1d-85d5-40da-b254-ebc3e65eff1f




## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Open a site that contains media files
* Open playlist web ui
* See if you have a "Add media from page" button on the bottom
* When clicking the button, playlist bubble in location bar should show up
